dashboard/app/jobs/cap/lockout_job.rb

```diff
@@ -0,0 +1,23 @@
+module CAP
+  class LockoutJob < ApplicationJob
```

Comment:
```suggestion
  # Attempts to transition the User to the locked-out state if they
  # should no longer be able to access Code.org due to their State's 
  # Child Account Policy.
  class LockoutJob < ApplicationJob
```
Consider adding some documentation so other devs can quickly understand the purpose of this job.